src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,646 @@
+
+### `none`
+
+<img src="{{ 'images/v23.2/migrations/lms_none.svg' | relative_url }}" alt="MOLT LMS shadowing mode - none" />
+
+`shadowMode: none` disables shadowing.
+
+- The LMS sends application requests to the source of truth only.
+- Query results from the source of truth are returned to the application.
+- Writes must be manually replicated from the source database to the target database.
+
+You can use this mode to perform a [consistent cutover](#consistent-cutover), along with another tool such as [change data capture (CDC)]({% link {{ page.version.version }}/cdc-queries.md %}) that replicates writes to the target database. {% comment %}For an example, see [Consistent cutover without shadowing](#consistent-cutover-without-shadowing).{% endcomment %}
+
+### `async`
+
+<img src="{{ 'images/v23.2/migrations/lms_async.svg' | relative_url }}" alt="MOLT LMS shadowing mode - async" />
+
+`shadowMode: async` writes to both databases.
+
+- The LMS sends application requests to the source of truth and target database in asynchronous threads, and waits only for the source of truth to respond.
+- Query results from the source of truth are returned to the application.
+- If an asynchronous request has not yet completed, subsequent asynchronous requests will be dropped.
+
+You can use this mode to confirm that your queries succeed on CockroachDB without verifying performance or correctness.
+
+{{site.data.alerts.callout_info}}
+`async` mode is intended for testing purposes.
+{{site.data.alerts.end}}
+
+### `sync`
+
+<img src="{{ 'images/v23.2/migrations/lms_sync.svg' | relative_url }}" alt="MOLT LMS shadowing mode - sync" />
+
+`shadowMode: sync` writes to both databases.
+
+- The LMS sends application requests to the source of truth and the target database, and waits for each to respond.
+- Query results from the source of truth are returned to the application.
+- Query results from the non-source of truth are discarded.
+
+{% comment %}
+You can use this mode to perform an [immediate cutover](#immediate-cutover).
+{% endcomment %}
+
+### `strict-sync`
+
+<img src="{{ 'images/v23.2/migrations/lms_strict-sync.svg' | relative_url }}" alt="MOLT LMS shadowing mode - strict-sync" />
+
+`shadowMode: strict-sync` writes to both databases and enforces correctness on both databases.
+
+- The LMS sends application requests to the source of truth and the target database, and waits for each to respond.
+- Query results from the source of truth are returned to the application.
+- If the non-source of truth returns an error, that error is returned instead of the query result from the source of truth.
+- If both databases return an error, the non-source of truth will return the source-of-truth error.
+
+{% comment %}
+You can use this mode to perform an [immediate cutover](#immediate-cutover).
+{% endcomment %}
+
+## Perform a cutover
```

Comment:
@ZhouXing19 do we support automatic consistent cutover here? If so, should we describe what that process entails?

Reply:
We don't support automatic flow currently as we still miss this part: https://github.com/cockroachlabs/crdb-proxy/issues/539
This is the only missing part for automatic cutover, and It's on my plate for the Oct milestone